samples/csharp/getting-started/BinaryClassification_HeartDiseasePrediction/HeartDiseasePrediction-Solution/DataStructure/HeartData.cs

```diff
@@ -3,23 +3,6 @@
 namespace HeartDiseasePredictionConsoleApp.DataStructure
 {
```

Comment:
@MarcoZama  I think we need features such as Age, Sex, CP,TestBPS…. etc when training the model. Can you include them also in HeartData class.

Reply:
But in the heartdata class are presents theese features. before I just remove  the heartdataimport class.

Reply:
@prathyusha12345 please can you explain me? i think i don't understand.

Reply:
whatever the column we are using for features ,place those columns in a class.For example in this case the class is HeartData. Hope you understand and thanks for wrapping them in HeartData class.